shared/login/forms/intro.render.native.js

```diff
@@ -0,0 +1,12 @@
+import React, {Component} from 'react'
+import {View, Text} from 'react-native'
+
+export default class Render extends Component {
+  render () {
+    return (
+      <View>
+        <Text>TODO 123</Text>
```

Comment:
@chrisnojima When I change this, the packager doesn't pick it up unless I restart the packager.